src/backend/cgcod.c

```diff
@@ -726,7 +726,7 @@ code *prolog()
     assert(usedalloca != 1);
     AllocaOff = alignsection(usedalloca ? (Foff - REGSIZE) : Foff, REGSIZE, bias);
 
-    CSoff = alignsection(AllocaOff - cstop * REGSIZE, REGSIZE, bias);
+    CSoff = alignsection(AllocaOff - (int)cstop * REGSIZE, REGSIZE, bias);
```

Comment:
> As explained above, there is no bug to be noticed easily.

Actually, there might be some, but I wasn't able to trigger execution of cases of operand flavor `FLcs`, `FLndp` or `FLlocalsize` in `assignaddrc()`.